rails/app/models/kaiju/project_json.rb

```diff
@@ -33,7 +33,7 @@ def self.as_json_lite(id, base_url, _options = {})
       project = klass.by_id(id)
       hash = project.as_json(only: %i[name owner update_date_time type])
       hash['owner'] = UserJson.map_id(hash['owner']) { |owner_id| UserJson.as_json(owner_id, base_url, lite: true) }
-      hash['workspace_count'] = project.workspaces.count
+      hash['workspace_count'] = WorkspaceJson.map_ids(project.workspaces.value).count
```

Comment:
This might qualify for a change log update, but doing so would probably break the other [PR](https://github.com/cerner/kaiju/pull/61)